packages/aws-cdk-lib/aws-ses/lib/dedicated-ip-pool.ts

```diff
@@ -49,6 +70,7 @@ export class DedicatedIpPool extends Resource implements IDedicatedIpPool {
 
     const pool = new CfnDedicatedIpPool(this, 'Resource', {
       poolName: this.physicalName,
+      scalingMode: props.scalingMode ?? ScalingMode.STANDARD,
```

Comment:
```suggestion
      scalingMode: props.scalingMode,
```
I think we should let CloudFormation handle the default in this case.